### ISSUE TITLE
Adding Graphql branch

### ISSUE THREAD

**Issue by ramyak-mehra** [shubhank-saxena/djangoReact-template#1]
It would be nice to have different kind of branches serving basic things like authentication user-management etc in rest or graphql

**Comment by shubhank-saxena** [comment]
Yes! This is the aim! Feel free to develop a new branch of whatever you feel like!

**Comment by ramyak-mehra** [comment]
I am trying to add jwt authentication for graphql , is there any different way of configuring middlewares in this project because i cant seem to add one. Used the official documentation [here](https://django-graphql-jwt.domake.io/en/latest/quickstart.html#installation). I have used this in a couple of projects works just fine .
Problem i am facing:
It is unable to introspect the schema after adding a middleware.
![image](https://user-images.githubusercontent.com/55009050/92909814-175b4680-f445-11ea-8523-84afbe797cf7.png)
Graphql Works just fine otherwise

**Comment by shubhank-saxena** [comment]
Nothing specific with project-wise. Try installing graphene. Maybe that helps

**Comment by ramyak-mehra** [comment]
> Nothing specific with project-wise. Try installing graphene. Maybe that helps

I have added and configured graphene and its working fine 
I will look into it

**Comment by ramyak-mehra** [comment]
@shubhank-saxena  make a new branch for graphene in the main repo

**Comment by shubhank-saxena** [comment]
Yep. Now it makes sense😂

**Comment by shubhank-saxena** [comment]
@ramyak-mehra I have added a graphql branch. You can now rebase your branch on that
